### DISCUSSION TITLE
Button-Links in Table-View

### DISCUSSION THREAD

**Discussion by lucameusburger** [directus/directus#13342] [Feature Requests]
It would be nice to select buttons in the table/list-view. Fields such as group or accordion are not selectable in the table-view, however button-links are also not selectable. would be nice to have buttons in the table-view.

For example:
I made a link-button that points to a custom endpoint that generates a pdf for the given item. Would be really nice to have this button in the table-view, so you don not have to click each item, find the button, click, go back, ...

Would look something like this:
<img width="908" alt="image" src="https://user-images.githubusercontent.com/25350597/168681858-49915dd9-6a58-485a-9c46-413fcb0c7435.png">

**Comment by benhaynes** [comment]
I agree with this! Selecting an alias button field in the list view would be an intentional override of the row click. 👍

**Reply by rijkvanzanten** [reply]
Still gotta be careful with the whole click on click problem we're already facing in some of the other displays though 👀

**Reply by lucameusburger** [reply]
I made a custom display extension for making tel:, mailto: and normal hrefs clickable. It works perfectly fine and is basically just:

    <template>
	    <a class="display-link" v-on:click.stop="handleLink()">{{value}}</a>
    </template>

    <script>
    export default {
	    props: {
	    	value: String,
	    },
	    methods:{
		    handleLink(){
			    window.open(this.value);
		    }
	    }
        };
    </script>

<img width="1501" alt="Unbenannt-1" src="https://user-images.githubusercontent.com/25350597/168846214-d153730c-80d1-4d10-be08-1902f188cd0c.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
